src/client/components/network-editor/controller.js

```diff
@@ -32,6 +32,47 @@ export class NetworkEditorController {
     this.drawModeEnabled = false;
   }
 
+  /**
+   * Replaces the current network with the passed one.
+   * @param {Object} [elements] Cytoscape elements object
+   * @param {Object} [data] Cytoscape data object
+   * @param {Object} [style] Optional Cytoscape Style object
+   */
+  setNetwork(elements, data, style) {
+    this.cy.remove(this.cy.$().remove());
```

Comment:
```suggestion
    this.cy.elements().remove();
```

You have an extra remove call here.  You can just do one on the elements.